tutorials/customizing-base-map-styles/index.md

```diff
@@ -0,0 +1,379 @@
+---
+title: Customizing base map styles
+description: If you are getting bored of looking at the same two default basemaps in Earth Engine, or need to use a specific visualization for functionality/convenience, this tutorial will show you how to edit all of its characteristics.
+author: TC25
+tags: Earth Engine, Base Map, UI, Widgets
+date_published: 2019-09-16
+---
+
+[Open In Code Editor](https://code.earthengine.google.com/283c86dd3189788452b5fb64174a1b43)
+
+The user is going to learn how to change the options of the map object to create new base maps
+
+## The default maps in Earth Engine
+
+Earth Engine's base maps are those in Google's Map API. The default options include:
+
+- roadmap, which displays the default road map view,
+- satellite, which displays Google Earth satellite images,
+- hybrid, which displays a mixture of normal and satellite views, and
+- terrain, which displays a physical map based on terrain information.
+
+## Changing the basic map style
+
+We can start by changing the style of the base map. One easy fix is to invert the lightness to get a darker background, like so:
+
+```javascript
+
+var baseChange = [
+  "stylers": [ {"invert_lightness": true} ]
+]
+Map.setOptions("baseChange",{"baseChange":baseChange})
+
+```
+The main styler options include:
+
+ - hue: indicates the basic color
+ - lightness:indicates percentage change in brightness of element
+ - saturation: indicates percentage change in basic color of element
+ - gamma: Gamma correction of the element (0.01 and 10.0)
+ - invert_lightness: inverts the existing lightness
+ - visibility: Changes visibility options for an element (on, off, or simplified)
+ - color: Sets the color of elements (using RGB Hex Strings)
+ - weight: Sets the weight of a feature in pixels
+
+
+## Changing map elements
+
+The google maps API (and by extension, Earth Engine) gives you the ability to control a huge number of map features and elements
+
+The full list of features that you can modify:
```

Comment:
I think it's fine to show a few examples here, but it's better to link to the canonical documentation (https://developers.google.com/maps/documentation/javascript/style-reference, which you point at below) instead of recreating that documentation here.

If Google Maps API adds/removes/changes options, our docs will be stale.

Reply:
Changed the language a bit so as to not repeat the Google Maps documentation.